hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/audit/parser/common/DatabaseHelper.java

```diff
@@ -93,8 +92,9 @@ private static void loadProperties() {
 
   private static boolean createAuditTable(String dbName) throws Exception {
     try (Connection connection = getConnection(dbName);
-         Statement st = connection.createStatement()) {
-      st.executeUpdate(properties.get(ParserConsts.CREATE_AUDIT_TABLE));
+        PreparedStatement st = connection.prepareStatement(
```

Comment:
Since the query does not take any params, a SQL injection cannot happen here.

Reply:
Thanks for review, I will roll back this modification,PTAL.